### PR TITLE
initial deployment

### PR DESCRIPTION
**Pull request by whoburg**
work in progress, running through the checklist at https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/

### CODE REVIEW

**Comment by whoburg** [comment]
@matthewdominick, this is ready for review. (click on "Files changed" above, and then start a review). If ready to merge, remember to do so using "Rebase and merge" from the dropdown.

File: feedbacksite/feedbacksite/settings.py [hunk 1]
```diff
@@ -20,13 +20,18 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = ')-z3%sv#3ql+)cimzk&^n*vwodvqdy8e^p&btkugue5=ur9j@u'
```
Comment (matthewdominick) [inline feedbacksite/feedbacksite/settings.py]: Glad you put this in a text file.  What is the Secret Key used for?

Comment (whoburg) [reply feedbacksite/feedbacksite/settings.py]: crypto signing to detect tampering. see more: https://docs.djangoproject.com/en/2.1/topics/signing/

**Comment by whoburg** [comment]
looks like rebase-merge did not work here? I'm going to attempt to rebase master and force-push.

**Comment by whoburg** [comment]
never mind -- it looks good. my fault.

**Pull request merged**